src/lib/button-toggle/button-toggle.scss

```diff
@@ -92,7 +92,9 @@ $mat-button-toggle-border-radius: 2px !default;
   }
 }
 
-.mat-button-toggle-ripple {
+// Increase specificity because ripple styles are part of the `mat-core` mixin and can
+// potentially overwrite the absolute position of the container.
+.mat-button-toggle .mat-button-toggle-ripple {
```

Comment:
Consider nesting this under the `.mat-button-toggle` selector. Same goes for the other changed child selectors in this PR.

Reply:
I considered doing that, but went with that approach because I wanted to separate the ripple stuff as much as I can from the actual component styles.